README.md

```diff
@@ -161,7 +142,12 @@ available to users.
 
  * **`splitCell`**`(state: EditorState, dispatch: ?fn(tr: Transaction)) → bool`\
    Split a selected cell, whose rowpan or colspan is greater than one,
-   into smaller cells.
+   into smaller cells. Use the first cell type for the new cells.
+
+
+ * **`splitCellWithType`**`(getType: fn({row: number, col: number, node: Node}) → NodeType) → fn(EditorState, dispatch: ?fn(tr: Transaction)) → bool`\
```

Comment:
can it stay private?

Reply:
Well the main idea is to be able to use it externally like this:

```javascript
splitCellWithType(({ row, col }: { row: number; col: number }) => {
    if (
      (row === 0 && withRowHeader) ||
      (col === 0 && withColumnHeader)
    ) {
      return tableHeader;
    }

    return tableCell;
  })
```

This delegate the decision about what cell type should be used to the consumer.